src/common/search-engines.ts

```diff
@@ -302,4 +309,52 @@ export const SEARCH_ENGINES: Readonly<Record<SearchEngineId, Readonly<SearchEngi
       name: 'searchEngines_startpageName',
     },
   },
+  searx: {
+    contentScripts: [
+      {
+        matches: [
+          'https://*.searx.be/*',
```

Comment:
Make the match patterns more narrow. Is access for every subdomain and page on `searx.be` really needed? Wouldn't `https://searx.be/search` (POST) and `https://searx.be/search?*` (GET) be enough?

The same applies to the following match patterns.

Reply:
What would be the difference or benefit from that? (I ask honestly)

Reply:
done, I will only use the POST method since the GET one would defeat the purpose of using Searx (privacy)